Sources/Plasma/NucleusLib/pnNetBase/pnNbConst.h

```diff
@@ -43,6 +43,8 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #ifndef pnNbConst_inc
 #define pnNbConst_inc
 
+#include "HeadSpin.h"
```

Comment:
This is a very lightweight constant only file. I think it would be better to avoid adding this and just using `unsigned` like the rest of the file.